app/refactor/viewmodels/p5/tad/Table2ViewModel.scala

```diff
@@ -68,7 +68,7 @@ object Table2ViewModel {
         _.flatMap(_.AdditionalInformation).map(_.asString)
       ).take100,
       guarantees = ie029.Guarantee.map(_.asString).semiColonSeparate.take100,
-      authorisations = ie029.Authorisation.map(_.asString).toBeContinued().take100
+      authorisations = ie029.Authorisation.map(_.asString).semiColonSeparate.first3
```

Comment:
first3 seems a bit overcomplicated if I've understood it correctly...

Is this not just `authorisations = ie029.Authorisation.take(3).map(_.asString).semiColonSeparate` ?

Reply:
Think there'll still be a better way of handling that than by joining them with `;` and then splitting them with `;`

Reply:
How about

```
authorisations = ie029.Authorisation.map(_.asString).take3.semiColonSeparate
```

where take3 is inside `RichStringSeq`

```
    private def takeN(n: Int): Seq[String] =
      if (value.length > n) {
        value.take(n) ++ Seq("...")
      } else {
        value.take(n)
      }
    
    def take3: Seq[String] = takeN(3)
```

Reply:
Would

```
authorisations = ie029.Authorisation.map(_.asString).take3(_.semiColonSeparate)
```

```
    private def takeN(n: Int)(f: Seq[String] => String): String =
      if (value.length > n) {
        f(value.take(n)) + "..."
      } else {
        f(value.take(n))
      }

    def take3(f: Seq[String] => String): String = takeN(3)(f)
```